shared-plugins/two-factor/class-two-factor-core.php

```diff
@@ -93,7 +93,7 @@ class Two_Factor_Core {
 	 * @since 0.1-dev
 	 */
 	public static function add_hooks( $compat ) {
-		add_action( 'plugins_loaded', array( __CLASS__, 'load_textdomain' ) );
+		add_action( 'init', array( __CLASS__, 'load_textdomain' ) );
```

Comment:
Technically, this could go into `after_setup_theme` but according to

https://github.com/WordPress/wordpress-develop/blob/522a9091c3a48553e61800de1967c7d77ff26e1c/src/wp-includes/l10n.php#L1002-L1013

> Attempted to load translations for the `xxx` domain too early. Translations should be loaded after the `after_setup_theme` action has fired, to ensure that the current user is already set up.

Current user is set after `after_setup_theme`; that's why I have chosen `init`.